### DISCUSSION TITLE
Motion detection stops working on H12, firmware 022070

### DISCUSSION THREAD

**Discussion by Reflexeye** [gopro/labs#967] [Q&A] [answered]
Hi - video 2.7k, 200fps, no auto offs set, no duration limit. Settings scene - start, stop,delay,hold all on 5 (mask un-set as only works in older Heroes). However, motion detection it seems randomly stopping. Any guidance much appreciated.

**Comment by dnewman-gpsw** [comment]
More info. "Randomly stopping", please explain.  Does it fail at other frame rates?  Share the exact command(s) you are using and any other Labs extensions applied.

**Reply by Reflexeye** [reply]
After a varying number of successful triggers (e.g. between 5 and 20) the motion detection function stops (or rather clears from the camera - no longer reporting 'differences' on screen. Commands are as above, all set at 5 (attached qr image). I'm only using motion control extension. I tried at 4K but then stopped as I'd read in discussions that 4K is unreliable and it behaves similarly - namely stopped functioning after a limited number of captures.
![Screenshot_20240619-184733](https://github.com/gopro/labs/assets/167014918/66cc55f0-2e7f-447f-a008-ed8edda84b63)

**Reply by dnewman-gpsw** [reply, marked as answer]
Okay, this was recently discussed, and not new.

Try ```!SM5D5H5!1R```  Just type it in the added command, don't use the QRControl UI for motion detection. 

This is a bug on H12, that will be fixed in the next release, but he workaround above is reliable.

**Reply by Reflexeye** [reply]
I searched discussions but didn't find. However - thank you for the info.
